libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -131,10 +145,11 @@ class SolanaWeb3Manager {
     if (feePayerAddress) {
       this.feePayerAddress = feePayerAddress
       this.feePayerKey = SolanaUtils.newPublicKeyNullable(feePayerAddress)
-    } else if (feePayerKeypairs && feePayerKeypairs.length) {
-      this.feePayerAddress = feePayerKeypairs[0].publicKey
+    } else if (feePayerKeypairs?.length) {
+      // @ts-expect-error this seems wrong
+      this.feePayerAddress = feePayerKeypairs[0]?.publicKey
```

Comment:
Surprised it has an error here

Reply:
yeah not sure what i was thinking there, fixed